### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Jacobs Arthur and Iseni Aladin

### DISCUSSION THREAD

**Discussion by Arthur2479** [heig-vd-dai-course/heig-vd-dai-course#385] [Show and tell]
We'd like to develop a picture editing software. Here are the features we'd like to implement :

- Convert a picture to B&W
- Blur a picture or part of it (if given optional coordinates)
- Crop the picture to a new format
- Invert the colors
- Apply a color filter (Instagram-like)
- Reduce a picture's resolution

We are conscious this may be a bit too many functionalities. They are sorted from favorite to least favorite, so if that's too much we'll just skip the last ones.
On the other hand, if you think this is not enough, we have a whole lot more ideas! 😄 

UPDATE :

Here's the repo link: https://github.com/HEIG-VD-DAI-Iseni-Jacobs/pictures-cli-editor
And the hash: ca38401e9de6f9821ea1cf84c0cda84ea34401f8

**Comment by ludelafo** [comment]
It's more than enough for me, thank you! Even if you manage to implement "only" the first two-three elements of the list, that's OK for me. 

@GeraudSilvestri, do you agree?

**Reply by GeraudSilvestri** [reply]
I agree as well, it is a lot of functionalities and as @ludelafo said during class an app that does a few things perfectly is better than one that does a lot badly. So focus on a few of them and once those are done think about doing another.

**Reply by Arthur2479** [reply]
Noted. We'll focus on the first ones then!

**Comment by Arthur2479** [comment]
@aladin-heig

**Comment by Arthur2479** [comment]
Here's the repo link: https://github.com/HEIG-VD-DAI-Iseni-Jacobs/pictures-cli-editor
And the hash: ca38401e9de6f9821ea1cf84c0cda84ea34401f8

**Reply by Arthur2479** [reply]
We did the first three features :)
